lib/std/special/compiler_rt/emutls.zig

```diff
@@ -376,19 +376,19 @@ test "__emutls_get_address with default_value" {
 
 test "test default_value with differents sizes" {
     const testType = struct {
-        fn _testType(comptime T: type, value: T) void {
+        fn _testType(comptime T: type, value: T) !void {
             var def: T = value;
             var ctl = emutls_control.init(T, &def);
             var x = ctl.get_typed_pointer(T);
             try expect(x.* == value);
         }
     }._testType;
 
-    testType(usize, 1234);
-    testType(u32, 1234);
-    testType(i16, -12);
-    testType(f64, -12.0);
-    testType(
+    try testType(usize, 1234);
```

Comment:
Can we get a bit more coverage for this test? Right now it's only running when targeting OpenBSD and thus prone to random breakage.